includes/payment-tokens/class-wc-stripe-payment-tokens.php

```
@@ -151,14 +157,152 @@ public static function customer_has_saved_methods( $customer_id ) {
+			return $tokens;
+		}
+
+		$gateway = WC_Stripe::get_instance()->get_main_stripe_gateway();
+
+		[ 'stored' => $stored_tokens, 'deprecated' => $deprecated_tokens ] = $this->categorize_existing_tokens( $tokens );
+
+		$active_reusable_types = $this->get_active_reusable_payment_method_types();
+		$customer              = new WC_Stripe_Customer( $customer_id );
+
+		try {
+			$payment_methods    = $customer->get_all_payment_methods( $active_reusable_types );
```

PHPMD (2.15.0)

[error] The variable $payment_methods is not named in camelCase. (undefined)

(CamelCaseVariableName)